config/routes.rb

```diff
@@ -1,6 +1,3 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get 'random_greeting', to: 'greetings#random_greeting'
 end
```

Comment:
- [ ] Great job on your frontend as it is set up well and API is working fine however you need to configure cors for your api results to be fetched over any other ports like for example for your localhost port 3001  or production site etc. See example:

```rb

Rails.application.config.middleware.insert_before 0, Rack::Cors do
    allow do
        origins "http://localhost:3000"
        resource "*", headers: :any, methods: [:get, :post, :put, :patch, :delete, :options, :head], credentials: true
      end
  
      allow do
        origins "http://localhost:3001"
        resource "*", headers: :any, methods: [:get, :post, :put, :patch, :delete, :options, :head], credentials: true
      end
end
```
Also you need to uncomment and install the `gem "rack-cors"` from your gemfile.